docs/preview/specification/queries.md

```diff
@@ -174,11 +174,17 @@ VALUES (
 
 INSERT INTO ducklake_snapshot_changes (
     snapshot_id,
-    snapshot_changes
+    snapshot_changes,
+    author,
+    commit_message,
+    commit_extra_info
 )
 VALUES (
     ⟨SNAPSHOT_ID⟩,
-    ⟨CHANGES⟩
+    ⟨CHANGES⟩,
+    (AUTHOR),
```

Comment:
These should use angle brackets, i.e. ⟨...⟩ instead of (...). The syntax highlighter detects angle brackets and turns them into red-italic highlights.